setup.py

```diff
@@ -40,6 +40,7 @@
         'gunicorn',
         'kubernetes',
         'oauth2client==3.0.0',
+        'openstacksdk',
```

Comment:
pls duplicate it for dev (piplock file)

Reply:
you can do it with "pipenv install --dev"

Reply:
https://docs.pipenv.org/ you can read about it here
pipenv is recommended way to manage virtual environments

Reply:
@katyafervent @naumvd95 there are no piplock files in the repository